src/components/TabBar/TabBar.css.ts

```diff
@@ -0,0 +1,69 @@
+import styled from '../styled'
+import Toolbar from '../Toolbar'
+
+import { getColor } from '../../styles/utilities/color'
+import baseStyles from '../../styles/resets/baseStyles.css'
+
+const getAlignment = align => {
+  switch (align) {
+    case 'center':
+      return 'center'
+    default:
+      return 'flex-start'
+  }
+}
+
+const getDirection = align => {
+  switch (align) {
+    case 'right':
+      return 'row-reverse'
+    default:
+      return 'row'
+  }
+}
+
+export const TabBarUI = styled('nav')`
+  --BlueConfigGlobalFontSize: 14px;
```

Comment:
@plbabin We have a handy font util now to allow you to set the font-sizes for things:

Example:
https://github.com/helpscout/hsds-react/blob/82b707b52548e0feeb522963472c99eaae707f36/src/components/GreeterCard/GreeterCard.css.ts#L38

I would use this in this `.css.ts` file 😎  (within `SecContentUI` as well)

Reply:
fixed in b651d7f